Triangulation_2/doc/Triangulation_2/CGAL/Constrained_Delaunay_triangulation_2.h

```diff
@@ -230,7 +242,18 @@ template <class PointIterator, class IndicesIterator>
 std::size_t insert_constraints(PointIterator points_first, PointIterator points_last,
                                IndicesIterator indices_first, IndicesIterator indices_last);
 
+/*!
+Same as insert_constraint(points_first, points_last, indices_first, indices_last)` except that duplicated pair of points are only considered once.
```

Comment:
Looks like a sole backtick at the end of the line
Found by: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.2-Ic-14/logs_master/Triangulation_2.log, rest might be collateral damage.